lib/osv/models.py

```diff
@@ -442,6 +444,7 @@ def update_from_vulnerability(self, vulnerability):
 
     if vulnerability.HasField('modified'):
       self.last_modified = vulnerability.modified.ToDatetime()
+      self.import_last_modified = vulnerability.modified.ToDatetime()
```

Comment:
This unfortunately won't work. 

This is usually set after we do this: 
https://github.com/google/osv.dev/blob/61afe8fc2aa9b1a0bc17aad4b4f517a96b29c526/lib/osv/impact.py#L538

So it'll just end up being the same as `modified` after we modify it

We should instead save the modified date here (before we call `_analyze_vulnerability`)
https://github.com/google/osv.dev/blob/27eab62add6e0893b724c7fa2e963ca1e6c07494/docker/worker/worker.py#L451

and then set it here on the Bug directly:

https://github.com/google/osv.dev/blob/27eab62add6e0893b724c7fa2e963ca1e6c07494/docker/worker/worker.py#L477